platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java

```diff
@@ -27,11 +32,17 @@
 
 public class UnavailableUrlsTest {
 
+    private static final String MAX_TIMEOUT_SECONDS = "300";
+
+    private static final String INITIAL_TIMEOUT_SECONDS = "10";
+
     private ScheduledExecutorService scheduler;
 
     @Before
     public void setUp() throws Exception {
         scheduler = Mockito.mock(ScheduledExecutorService.class);
+        System.setProperty(MAX_TIMEOUT_SECONDS_PROPERTY, MAX_TIMEOUT_SECONDS);
+        System.setProperty(INITIAL_TIMEOUT_SECONDS_PROPERTY, INITIAL_TIMEOUT_SECONDS);
```

Comment:
I'd like to do that I looked around on the web and only found two options:
1. Use PowerMock to mock out the System class.
2. Clean up after each test with @After. 

I think option #2 is the lesser of two evils. Making the change now.